package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
+    "analyze": "webpack-bundle-analyzer dist/golden-sample-app/stats.json",
     "ng": "nx",
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
-    "start": "nx serve",
-    "build": "nx build --configuration=production",
+    "start": "ng serve",
+    "start:docker": "docker run -p 8080:8080 golden-sample-app",
     "test": "nx test --code-coverage",
+    "build": "nx build --configuration production",
+    "build:stats": "ng build --configuration production --stats-json",
```

Comment:
Why this one is not using `nx build...` ?

Reply:
To use proxy config for that build